core/types/block_ext_test.go

```diff
@@ -0,0 +1,188 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package types
+
+import (
+	"math/big"
+	"reflect"
+	"testing"
+	"unicode"
+	"unsafe"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/libevm/pseudo"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCopyHeader(t *testing.T) {
```

Comment:
Even when using libevm's `CopyHeader` next (#759), this test should be kept to ensure our PostCopy hook is coded correctly. `CopyHeader` can have subtle yet important changes and this test would help debunk some imo.

Reply:
agree about keeping the test, but we may need to find a better home for it soon.

Reply:
What do you mean by better home 🏠 ? 🤔